docs/man.rst

```diff
@@ -207,17 +208,11 @@ The following commands are supported in all interactive modes:
    Do not commit these changes, instead leaving them staged in the index.
    Index lines must come last in the file.
 
-   .. note:
-      Commits deleted from the to-do list are treated as an index action. To
-      completely discard changes, additionally use :manpage:`git-reset(1)` to
-      discard the changes to the index.
```

Comment:
Let's keep this note but explicitly say "Commits may not be deleted from the to-do list. To remove a commit, mark it as an `index` action, and use git-reset(1) to discard changes to the index."